jib-gradle-plugin/CHANGELOG.md

```diff
@@ -11,6 +11,7 @@ All notable changes to this project will be documented in this file.
 
 - Removed deprecated `jib.jvmFlags`, `jib.mainClass`, `jib.args`, and `jib.format` in favor of the equivalents under `jib.container` ([#461](https://github.com/GoogleContainerTools/jib/issues/461))
 - `jibExportDockerContext` generates different directory layout and `Dockerfile`.
+- File timestamps in the build image are set to 1 second since the epoch (hence 1970-01-01T00:00:01Z); previously they were set to the epoch. ([#1079](https://github.com/GoogleContainerTools/jib/issues/1079))
```

Comment:
maybe add "to resolve compatibility with <Java 6 applications where file time of 0 means nonexistent"?